Integrations/illuminate/illuminate.yml

```diff
@@ -0,0 +1,520 @@
+description: Enriches Demisto indicators with illuminate REST API indicator data.
+configuration:
+- display: illuminate API Credentials (username/password)
+  name: credentials
+  defaultvalue: ''
+  type: 9
+  required: true
+- display: Domain of illuminate server to use
+  name: server
+  defaultvalue: illuminate.analystplatform.com
+  type: 0
+  required: true
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: 'false'
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: 'false'
+  type: 8
+  required: false
+script:
+  script: '-'
+  type: python
+  commands:
+  - name: domain
+    arguments:
+    - name: domain
+      required: true
+      description: The domain to query information for
+      default: true
+      isArray: true
```

Comment:
The API supports Arrays? If not you will need to address this at the code.
Same goes for all the other enrichment commands